app/javascript/dashboard/routes/dashboard/settings/inbox/PreChatForm/Settings.vue

```diff
@@ -47,8 +74,13 @@
 <script>
 import { mapGetters } from 'vuex';
 import alertMixin from 'shared/mixins/alertMixin';
-
+import draggable from 'vuedraggable';
```

Comment:
Should show that we have this feature, with a drag handle (Something like below).

<img width="661" alt="CleanShot 2022-03-10 at 20 28 17@2x" src="https://user-images.githubusercontent.com/15716057/157688930-af29f317-2550-4d0e-b27a-e5ab39a9adc2.png">